Koha/Plugin/Com/PTFSEurope/Pubmed/Api.pm

```diff
@@ -64,7 +64,7 @@ sub parse_to_ill {
             return join('; ', @auth_arr);
         },
         article_title => sub {
-            return shift->{title};
+            return shift->{vernaculartitle};
```

Comment:
Confirm this. It's likely to be just title and not vernaculartitle

Reply:
Did you manage to confirm this?

Reply:
Yes, it's title, fixed in 69a8400b9539d26ecdfd0b0f17258cb74a8cdf5a